kubernetes/ams-k8s-deployment-edge.yaml

```diff
@@ -45,6 +45,18 @@ spec:
 # You may also need to add -u and -p parameters for
 # specifying mongodb username and passwords respectively         
         args: ["-g", "true", "-s", "true", "-r", "true", "-m", "cluster", "-h", "mongo"]
+        livenessProbe:
+          httpGet:
+            path: / 
+            port: 5080
+          initialDelaySeconds: 30
+          periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 5080
+          initialDelaySeconds: 30
```

Comment:
It makes me think that initialDelaySeconds is high. It means that it will wait 30 seconds for all cases when a new pod spin up.

Reply:
Hi @mekya 

This is a time limit for pods with poor performance (starting java, connection db etc.). Nevertheless I can set it to around 10.

Reply:
OK. Let's make it 10 seconds. I'm confirmig the PR in advance. Then you'll change the `initialDelaySeconds`to 10  and merge the PR.